src/search.js

```diff
@@ -27,7 +32,7 @@ export function addNormalizedDescription(propertyName, objects) {
   return map(
     object => assoc(
       'normalizedDescription',
-      has(propertyName, object)
+      object[propertyName]
```

Comment:
Actually, it's more complicated than that… `Boolean(object[propertyName])` ≠ `has(propertyName, object)`, because [`has`](http://ramdajs.com/docs/#has) only looks for an **own** property and thus doesn't walk up the prototype chain.

So this change changes two things: the return type and the property scope.
I am personally fine with it, since the value is immediately consumed, and is so unambiguously as a boolean value. There is thus no API change at all, so no risk.
As a JS person, I even prefer the native approach, as I know exactly what looking up a property entails, while I had to look up ramda's doc to discover that `has` was an alias to `hasOwnProperty` (i.e. no prototype chain walk).

This comment, as well as [this one](https://github.com/openfisca/legislation-explorer/pull/70#discussion_r108659248) on initialising properties are clearly a matter of style and of embracing falsies or feeling reassured by pure boolean values.

If we are to all work on this code at one point or another, we should agree on the tooling and the coding style, such as embracing ramda or trying to get away from it. Unless we decide to get away from that library, I don't think we should change code that works.